apps/dashboard/src/DashboardApp.vue

```diff
@@ -394,13 +394,16 @@ export default {
 			}
 		},
 		enableStatus(app) {
-			this.enabledStatuses[app] = true
+			this.enabledStatuses.push(app);
 			this.registerStatus(app, this.allCallbacksStatus[app])
 			this.saveStatuses()
 		},
 		disableStatus(app) {
-			this.enabledStatuses[app] = false
-			const i = this.registeredStatus.findIndex((s) => s === app)
+			let i = this.enabledStatuses.findIndex((s) => s === app)
+			if (i !== -1) {
+				this.enabledStatuses.splice(i, 1)
+			}
+			i = this.registeredStatus.findIndex((s) => s === app)
```

Comment:
```suggestion
			const i = this.enabledStatuses.findIndex((s) => s === app)
			if (i !== -1) {
				this.enabledStatuses.splice(i, 1)
			}
			const j = this.registeredStatus.findIndex((s) => s === app)
```
And adjust the rest of the method.
I had to read it twice to realize why i was not a const. Slightly clearer like this IMO. We can afford declaring one more integer.